data/tutorial/04_contents.md

```diff
@@ -163,17 +162,16 @@ let add_car store car_number car =
     let info = Car_info.v "added %s" car_number in
     Car_store.set_exn store [car_number] car ~info
 
-let main =
+let () =
+    Eio_main.run @@ fun _ ->
     let config = Irmin_mem.config () in
-    let* repo = Car_store.Repo.v config in
-    let* t = Car_store.main repo in
-    let* () = add_car t "5Y2SR67049Z456146" car_a in
-    let* () = add_car t "2FAFP71W65X110910" car_b in
-    let+ car = Car_store.get t ["2FAFP71W65X110910"] in
+    let repo = Car_store.Repo.v config in
+    let t = Car_store.main repo in
+    let () = add_car t "5Y2SR67049Z456146" car_a in
```

Comment:
At the risk of losing consistency, getting rid of the `let () =` makes the direct-style code clearer, I think...